examples/config/ci/deployer.json

```diff
@@ -0,0 +1,153 @@
+{
+  "AWSProfile": "",
+  "AWSRegion": "us-east-1",
+  "AWSAvailabilityZone": "",
```

Comment:
Eventually, I think we'd want to try and use the same AZ, if possible, to contain costs unless it becomes a nightmare of flakiness. Trying other regions may help with that.